packages/docs/content/09-registries-and-packs/1-adding-a-custom-registry.md

```diff
@@ -55,7 +55,7 @@ For self-signed certificates, use the following command to generate certificates
 ```bash
 openssl req \
   -newkey rsa:4096 -nodes -sha256 -keyout tls.key \
-  -x509 -days 365 -out tls.crt
+  -x509 -days 1825 -out tls.crt
```

Comment:
@abhikb2005   The 365 day self-signed cert duration tends to cause outages down the road so suggested to generate as 5-year and really should change everywhere else in docs where default passwords, 1 year duration or even < 4096 private may be in use.